routes/order_verification.js

```diff
@@ -0,0 +1,44 @@
+/* Order Verification
+
+
+ Order Verification Pseudo Code
+
+ Receive package, if package contents = order contents then set status
+   of package to "Good" or "Verified", then send package out to shipping
+
+	const order = [{
+	item: "Toy",
+	qty: 2}
+	];
+
+	const package = {
+	item: "Toy", qty: 2 
+	};
+	
+
+	function verify () {
+		if ( order[i].item = package[i].item && order[i].qty = package[i].qty ) {
```

Comment:
Also, this pseudo code indicates that the order of the **order** and **package** const's *have* to be in the identical order. That can become problematic because you are not implicitly suggesting the order actually matters, but your conditional is testing for it.

This may require a bit more thought to work through, but an elegant solution can be discovered.